src/frontend/Komponenter/Behandling/TidligereVedtaksperioder/TidligereVedtaksperioderInfo.tsx

```diff
@@ -40,16 +48,80 @@ const FinnesTidligereVedtaksperioder: FC<{
     );
 };
 
+const TabellTidligereVedtaksperioder: React.FC<ITidligereVedtaksperioder> = ({
+    infotrygd,
+    sak,
+}) => (
+    <GridTabell kolonner={3}>
+        <>
+            <TabellVisning<{
+                stønad: Stønadstype;
+                verdi: { sak?: boolean; infotrygd?: boolean };
+            }>
+                ikon={TabellIkon.REGISTER}
+                tittel="Har bruker tidligere vedtaksperioder i EF Sak eller Infotrygd"
+                undertittel="(inkluderer kun EF VP, ikke PE PP)"
+                verdier={[
+                    {
+                        stønad: Stønadstype.OVERGANGSSTØNAD,
+                        verdi: {
+                            sak: sak?.harTidligereOvergangsstønad,
+                            infotrygd: infotrygd?.harTidligereOvergangsstønad,
+                        },
+                    },
+                    {
+                        stønad: Stønadstype.SKOLEPENGER,
+                        verdi: {
+                            sak: sak?.harTidligereBarnetilsyn,
+                            infotrygd: infotrygd?.harTidligereBarnetilsyn,
+                        },
+                    },
+                    {
+                        stønad: Stønadstype.SKOLEPENGER,
+                        verdi: {
+                            sak: sak?.harTidligereSkolepenger,
+                            infotrygd: infotrygd?.harTidligereSkolepenger,
+                        },
+                    },
+                ]}
+                kolonner={[
+                    {
+                        overskrift: 'Stønad',
+                        tekstVerdi: (d) => stønadstypeTilTekst[d.stønad],
+                    },
+                    {
+                        overskrift: 'Historikk i Infotrygd',
+                        tekstVerdi: (d) => mapTrueFalse(!!d.verdi.infotrygd),
+                    },
+                    {
+                        overskrift: 'Historikk i EF Sak',
+                        tekstVerdi: (d) => mapTrueFalse(!!d.verdi.sak),
+                    },
+                ]}
+            />
+        </>
+    </GridTabell>
+);
+
 const TidligereVedtaksperioderInfo: FC<{ tidligereVedtaksperioder: ITidligereVedtaksperioder }> = ({
     tidligereVedtaksperioder,
 }) => {
-    const { infotrygd } = tidligereVedtaksperioder;
-
-    if (!infotrygd) {
+    if (!tidligereVedtaksperioder.infotrygd) {
```

Comment:
Men hvis du bare har tidligereVedtaksperioder.sak så blir vel dette litt feil, eller?

Reply:
Hvis en person ikke har noe i infotrygd, fikk BT i vår og søker på nytt om BT i august - vil det ikke være "riktig" at det kun finnes "historikk" i ny løsning?

Reply:
Da vil det være grunnlagsdata på at det ikke er historikk i infotrygd. Alle behandlinger vil ha feltet infotrygd, men det vil være `false` på `harTidligereBarnetilsyn, harOver.... ` osv. Egentlig vet jeg ikke helt hvorfor `infotrygd`-feltet er nullable. Er jo morsobjektet som bør være det?

Reply:
Modellen er egentlige litt "feil" her.
Domeneobjektet inneholder infotrygd, og kanskje sak - for å være bakåtkompatibel
DTO'en blir mappet med nullable for begge, men at selve tidligereVedtaksperioder ikke er nullable.
Sånn att man får alltid tilbake en `tidligereVedtaksperioder` men der 
* infotrygd kan være null, 
* infotrygd har verdi, sak er null
* begge har verdi

Reply:
Hvis man kun har historikk i ny løsning, så har man false i alle verdier fra infotrygd, men at infotrygd fortsatt ikke er null

Reply:
Ja, burde egentlig gjort `tidligereVedtaksperioder ` nullable i dto'en

Reply:
Ja, mulig vi burde gjøre det 🤨 må vel nesten dele opp det i 3 steg då? 
* Endre frontend tidligereVedtaksperioder nullable
* Fikse backend
* Endre frontend til å sette infotrygd til not nullable

Reply:
Merger bare nå, så får vi vurdere å skrive om senere.